openquake/server/db/actions.py

```diff
@@ -391,15 +391,15 @@ def calc_info(calc_id):
     return response_data
 
 
-def get_calcs(request_get_dict, user_name, user_is_super=False, id=None):
+def get_calcs(request_get_dict, user_name, user_acl_on=False, id=None):
     """
     :returns:
         list of tuples (job_id, user_name, job_status, job_type,
                         job_is_running, job_description)
     """
     # helper to get job+calculation data from the oq-engine database
     jobs = models.OqJob.objects.filter()
-    if not user_is_super:
+    if user_acl_on:
         jobs = jobs.filter(user_name=user_name)
```

Comment:
Please add a comment here, like

user_acl_on is true if the configuration acl_on is True or if the user is a Django super user.